### PR TITLE
Add plugin list to README

### PR DESCRIPTION
**Pull request by willbarrett**
Fixes https://github.com/pushpop-project/pushpop/issues/22

Let me know what you think.

### CODE REVIEW

**Comment by josephwegner** [comment]
Could we move this down to the `Plugins` section? I'd like to keep the docs first, and let people scroll to find the list of current plugins. Perhaps change that intro line (deleted in this PR) to anchor link down to that section.

**Comment by willbarrett** [comment]
Sure. No problem.

**Comment by willbarrett** [comment]
Updated.

File: README.md [hunk 1]
```diff
@@ -363,9 +364,23 @@ unicorn -c unicorn.rb
 The community-driven [pushpop-recipes](https://github.com/pushpop-project/pushpop-recipes) repository contains jobs and templates
 for doing common things with Pushpop. Check it out for some inspiration!
 
+<a name="plugins"></a>
```
Comment (josephwegner) [inline README.md]: You actually don't need this line. GFM inserts the anchor text automatically.

**Comment by josephwegner** [comment]
Once that last comment is fixed, this is +1 and I will merge.

Thanks a bunch!

**Comment by willbarrett** [comment]
Huh - learn something new every day. Thanks!

Updated.

**Pull request merged**